modules/auth/model/cms_temporal_user.e

```diff
@@ -0,0 +1,44 @@
+note
+	description: "Summary description for {CMS_TEMPORAL_USER}."
+	date: "$Date$"
+	revision: "$Revision$"
+
+class
+	CMS_TEMPORAL_USER
+
+inherit
+
+	CMS_USER
+
+create
+	make,
+	make_with_id
+
+feature -- Access
+
+	personal_information: detachable STRING_32
```

Comment:
not at the moment, because user profile is not stored for now.
And we have to review CMS_USER, because it also have items, and it seems they represent a kind of profile for it, maybe I'm wrong.